composer.json

```diff
@@ -24,12 +24,12 @@
         "docs": "https://github.com/leanphp/phpspec-code-coverage#phpspec-code-coverage"
     },
     "require": {
-        "php": "~5.6||~7.0",
-        "phpspec/phpspec": "~3.0",
+        "php": "^7.0",
```

Comment:
Question: Is PHP 7.1 working with PHP Spec v4?

Reply:
Yes, it works with 7.1

Reply:
Then I suggest to add it too, just to make sure it won't be a problem for some projects in case they have new criteria like the minimum PHP version

Reply:
The `^7.0` should be specific enough as it will allow any 7.x versions to be used. The `^` means `not earlier version than`.